nntrainer/tensor/tensor.cpp

```diff
@@ -227,6 +241,31 @@ bool Tensor::operator==(const Tensor &rhs) const {
 #else
     throw std::invalid_argument("Error: enable-fp16 is not enabled");
 #endif
+  } else if (dim.getDataType() == ml::train::TensorDim::DataType::QINT8) {
+    const uint8_t *_data = getData<uint8_t>();
```

Comment:
we might need to check the zero point and scale as well.

Reply:
applied.